experimental/examples/imageviewer/iosApp/iosApp.xcodeproj/project.pbxproj

```diff
@@ -355,7 +355,7 @@
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_ASSET_PATHS = "";
-				DEVELOPMENT_TEAM = "${TEAM_ID}";
+				DEVELOPMENT_TEAM = 3CP3246D24;
```

Comment:
same here

Reply:
Thank you! Original snippet was restored.